Source/WebCore/css/CSSStyleRule.cpp

```diff
@@ -159,8 +159,11 @@ String CSSStyleRule::cssText() const
 
 void CSSStyleRule::cssTextForRules(StringBuilder& rules) const
 {
-    for (unsigned index = 0; index < length(); index++)
-        rules.append("\n  "_s, item(index)->cssText());
+    for (unsigned index = 0; index < length(); index++) {
```

Comment:
We should avoid calling `length()` for every loop iteration. We could also use `++index` since we're modifying this code as it is better practice when possible.